setup.py

```diff
@@ -44,6 +44,9 @@ def package_data(pkg, roots):
     entry_points={
         'xblock.v1': [
             'lti_consumer = lti_consumer:LtiConsumerXBlock',
+        ],
```

Comment:
Just a reminder to bump the version here 🙂

Reply:
I went for a big version bump since we're changing a lot of the internals and adding support for a new spec.